.github/workflows/pull-request-lint.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     permissions:
       pull-requests: write
     steps:
-      - uses: amannn/action-semantic-pull-request@v5.0.2
+      - uses: amannn/action-semantic-pull-request@v4.5.0
```

Comment:
Deze is wel wonderlijk

Reply:
Ik vermoed dat dat met https://github.com/GemeenteNijmegen/modules-projen/pull/6/ opgelost kan worden.